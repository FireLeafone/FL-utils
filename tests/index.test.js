const utils = require('../src');

describe('src func list', () => {
  it('export modules correctly', () => {
    expect(Object.keys(utils)).toMatchSnapshot();
  });
});
