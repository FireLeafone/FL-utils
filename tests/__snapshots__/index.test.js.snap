// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src func list export modules correctly 1`] = `
Array [
  "filterParams",
  "paramToObj",
  "queryString",
  "beNaN",
  "isArray",
  "isLength",
  "isNull",
  "isNumber",
  "isObject",
  "isString",
  "isUndefined",
  "maxArrNum",
  "minArrNum",
  "uniqueArray",
  "isRepeatArray",
  "arrayToTree",
  "arrayTreeCallBack",
  "deepCopy",
  "filter",
  "getNodeByKeyValues",
  "sortTree",
  "filterTree",
  "filterTreeFn",
  "debounce",
  "downloadLink",
  "drawLineArrow",
  "throttle",
  "antdFormFormat",
  "btSize",
  "formatDate",
  "numpoints",
  "hexToRgb",
  "rgbToHex",
  "unBindAntdForm",
  "emptyObj",
  "isEquivalentObj",
  "addNums",
  "randomId",
  "randomColor",
]
`;
